### ISSUE TITLE
Explore rewrite using gradle

### ISSUE THREAD

**Issue by ajhager** [ajhager/libgdx-sbt-project.g8#93]

**Comment by aginiewicz** [comment]
hmm... rewrite using gradle? can you share some light why such idea? I think that going #59 would be better for the Scala part - especially when sbt reaches 1.0 (the just released 0.13.7 gained some new nice features again, and it seems to be reaching maturity/stability).

Anyway I think that if we consider going gradle, it could be easier to add some changes to project generator from libgdx team to add scala support with things like https://github.com/saturday06/gradle-android-scala-plugin (should be easier than whole new template) and ask them to pull those changes in - then one could have a dropdown list in their gui with language. Especially, that Mario is exploring other languages (I think kotlin when I looked for the last time).

**Comment by ajhager** [comment]
Indeed, the only idea was to see if it is possible enjoy the benefit of work on the official workflow. I am still not sure if it is even possible.
